Assets/Player/CS_PlayerHealth.cs

```diff
@@ -0,0 +1,74 @@
+using JetBrains.Annotations;
```

Comment:
For what are you using Jet Brains Annotations?

Reply:
Tbh I dont really know what they are but its not grayed out so i didnt delete it, will do in class today and see what happens, we def not want unused dependencies